bin/next-start

```diff
@@ -14,9 +14,18 @@ const argv = parseArgs(process.argv.slice(2), {
     p: 'port'
   },
   boolean: ['h'],
-  default: { p: 3000 }
+  string: ['H'],
+  default: {
+    p: 3000,
+    H: '0.0.0.0'
```

Comment:
The Node Doc says `the server will accept connections on any IPv6 address (::) when IPv6 is available` I wonder if this default value works as expected.